R/data-storage-log-file.R

```diff
@@ -157,14 +178,19 @@ DataStorageLogFile <- R6::R6Class( # nolint object_name_linter
     # @name read_data
     # Reads the JSON log file
     # @param bucket string with path to file
-    read_data = function(bucket, date_from, date_to) {
+    read_data = function(
+      bucket, date_from, date_to, empty_template = dplyr::tibble()
+    ) {
       checkmate::assert_string(bucket)
       checkmate::assert_date(date_from)
       checkmate::assert_date(date_to)
 
-      json_log_msg <- readLines(bucket)
-      json_log <- dplyr::bind_rows(lapply(json_log_msg, jsonlite::fromJSON))
-      json_log %>%
+      if (!file.exists(bucket)) {
+        return(empty_template)
+      }
```

Comment:
I agree. I this could actually solve the issue I am having with the session logs.